tools/scripts/ValidateReleaseInfo.ps1

```diff
@@ -0,0 +1,98 @@
+# Constants
```

Comment:
Actual output of the script that appears in the PR logs:

Existing Releases
v1.1.0830.03
v1.1.0830.01
v1.1.0828.02
v1.1.0828.01
v1.1.0827.02
v1.1.0827.01
v1.1.0823.01
v1.1.0822.06
v1.1.0794.01
-----------------------
Checking Info for Production
Channel Info: @{current_version=1.1.0794.01; minimum_version=1.1.0794.01; installer_url=https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0794.01/AccessibilityInsights.msi; release_notes_url=https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Production/release_notes.md}
PASSED Does release v1.1.0794.01 exist?
PASSED Does release v1.1.0794.01 exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0794.01/AccessibilityInsights.msi exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Production/release_notes.md exist?
-----------------------
Checking Info for Insider
Channel Info: @{current_version=1.1.0828.01; minimum_version=1.1.0828.01; installer_url=https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0828.01/AccessibilityInsights.msi; release_notes_url=https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Insider/release_notes.md}
PASSED Does release v1.1.0828.01 exist?
PASSED Does release v1.1.0828.01 exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0828.01/AccessibilityInsights.msi exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Insider/release_notes.md exist?
-----------------------
Successfully validated release_info.json files